File: teste.java
class teste {
	static boolean primo(int num) {
		if (num < 2)
			return false;
		for (int i = 2; i < num; i++) {
			if (num % i == 0)
				return false;
		}
		return true;
	}

	public static void main(String[] args) {
		for (int i = 0; i < 100000; i++) {
			if (primo(i))
				System.out.println(i);
		}
	}
}

File: teste.dart
bool primo(num numero) {
  if (numero < 2) {
    return false;
  }
  for(num i = 2; i < numero; i++){
    if(numero % i == 0){
      return false;
    }
  }
  return true;
}

void main(){
  for(num i = 0; i < 100000; i++){
    if(primo(i)){
      print(i);
    }
  }
}
